jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4545,4 +4551,14 @@ public boolean isNestmateOf(Class<?> that) {
 	return nestMembers;
 }
 /*[ENDIF] Java11 */
+
+/*[IF Java12]*/
+	public Class<?> arrayType() {
+		throw new UnsupportedOperationException("Stub for Java 12 compilation");
+	}
+
+	public Optional<ClassDesc> describeConstable() {
+		throw new UnsupportedOperationException("Stub for Java 12 compilation");
+	}
+/*[ENDIF]*/
```

Comment:
I'd prefer that these closing directives were `/*[ENDIF] Java12 */` to match the style used in C/C++ code which makes it easier to associate with the opening directive.